platform/web/js/libs/library_godot_fetch.js

```diff
@@ -121,6 +125,10 @@ const GodotFetch = {
 				}
 				obj.reading = true;
 				obj.reader.read().then(GodotFetch.onread.bind(null, id)).catch(GodotFetch.onerror.bind(null, id));
+			} else if (obj.reader === null && obj.response.body === null) {
+				// Emulate a stream closure to maintain the request lifecycle
+				obj.reading = true;
+				GodotFetch.onread(id, { value: undefined, done: true });
```

Comment:
Added a period and set the comparison to use `==` instead of the strict one (`===`).

```suggestion
			} else if (obj.reader == null && obj.response.body == null) {
				// Emulate a stream closure to maintain the request lifecycle.
				obj.reading = true;
				GodotFetch.onread(id, { value: undefined, done: true });
```